.gitignore

```diff
@@ -0,0 +1,43 @@
+# Whisk
```

Comment:
I do not think this file is necessary to add.
Perhaps it is a mistake in the PR submission?